src/main/java/ru/neoflex/chmutenko/bank/CreditConveyor/service/ApplicationService.java

```diff
@@ -0,0 +1,76 @@
+package ru.neoflex.chmutenko.bank.CreditConveyor.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import ru.neoflex.chmutenko.bank.CreditConveyor.dto.LoanOfferDTO;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+import ru.neoflex.chmutenko.bank.CreditConveyor.service.util.UtilCalculator;
+
+
+@Service
+public class ApplicationService {
+
+    @Value("${loanOffer.baseRate}")
+    private BigDecimal baseRate;
+    @Value("${loanOffer.insurance}")
+    private BigDecimal insuranceAmount;
+
+    private final UtilCalculator calculator;
+    private static long applicationId;
+    private static final Logger logger = LoggerFactory.getLogger(ApplicationService.class);
```

Comment:
```suggestion

    private static final Logger logger = LoggerFactory.getLogger(ApplicationService.class);
    
    private static long applicationId;
    
    private final UtilCalculator calculator;
    
    @Value("${loanOffer.baseRate}")
    private BigDecimal baseRate;
    
    @Value("${loanOffer.insurance}")
    private BigDecimal insuranceAmount;
```
Сначала идут статические  финальные поля потом статические
потом файналы, потом стандартные